ca_description/urdf/create_apriltag.xacro

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" ?>
+<robot xmlns:xacro="http://ros.org/wiki/xacro">
+
+<xacro:macro name="create_apriltag">
+ <joint name="apriltag_joint" type="fixed">
+    <origin xyz="0 0 0.25" rpy="0 0 0" />
```

Comment:
In case I want to add this aprilTag in another place, I won't be able to do so.

In order to do that, create a parameter in this macro that sets this `origin` block.